src/helpers/time.ts

```
@@ -57,3 +57,25 @@ const fetchServerTime = async () => {
+
+export function calculateQFTimeDifferences(
+	providedTime: string | undefined,
+): number {
+	if (!providedTime) {
+		return 0;
+	}
+
+	const providedDate = new Date(providedTime);
+	const currentDate = new Date();
+
+	if (isNaN(providedDate.getTime())) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)